### ISSUE TITLE
An error occurred starting screensaver

### ISSUE THREAD

**Issue by RalfJung** [lxqt/lxqt#985]
After leaving my laptop alone for some time, when I come back and unlock my screen, I always get the following message:

```
An error occurred starting screensaver. Action 'activate' failed. Ensure you have xscreensaver installed and running.
```

I have xscreensaver installed, and I have it enabled in the lxqt session configuration. However, I don't actually want a screensaver, so I picked "Disable Screen Saver" in "Preferences - Screensaver". I just tried setting that to "Blank Screen Only" instead and will see whether that makes any difference.

Still, there should be no error from a legit configuration.

**Comment by RalfJung** [comment]
Even with the new configuration ("Blank Screen Only"), I am still seeing that error.

**Comment by RalfJung** [comment]
When interacting with xdg-screensaver directly, I can reproduce the error code 4 if xdg-screensaver is called when the screen is already locked. So I guess that's what is happening here.

**Comment by RalfJung** [comment]
I now simply disabled the idle watcher provided by LXQt, and rely on xscreensaver to lock the screen when the machine is idle. This also has more features, e.g. turning of the screen a few minutes before locking it.

**Comment by pmattern** [comment]
Setting "Mode: Disable Screen Saver" of XScreenSaver's GUI `xscreensaver-demo` just ensures that none of the available screensavers gets launched but doesn't stop the daemon binary `xscreensaver`.
The daemon can be prevented from starting at sessions' beginning by unchecking the box at `lxqt-config-session` Autostart - LXQt Autostart.
(Eventually turning the daemon into a so-called "LXQt Module" was addressed in #966.)

**Comment by Vladimir-csp** [comment]
disabling xscreensaver daemon will make screen locking impossible.

**Comment by alex3kov** [comment]
I also get this all the time when resuming from suspended state (systemctl suspend).

**Comment by alex3kov** [comment]
This is still happening on LXQt 0.12.0.

**Comment by agaida** [comment]
Should be solved with the latest powermanagement changes

**Comment by Raytle** [comment]
I'm running Lubuntu 20.04 LTS

1. I navigated to Menu » Preferences » LXQt Settings » Session Settings » Autostart
2. I unchecked the check box next to Xscreensaver
3. I also ran `sudo apt-get remove xscreensaver`

Each time I run `systemctl suspend` the following popup message appears on my screen...

> An error occurred starting screensaver. Action 'activate' failed. Ensure you have xscreensaver installed and running.

I don't want that popup message appears on my screen.
